backend/src/lib.rs

```diff
@@ -0,0 +1,27 @@
+#![allow(clippy::multiple_crate_versions)]
+
+pub mod connector;
+pub mod error;
+pub mod models;
+pub mod services;
+
+// [TEMP]
+#[allow(clippy::unused_async)]
+#[utoipa::path(
+        get,
+        path = "/",
+        responses(
+            (status = 200, description = "Hello Bob!")
+        )
+    )]
+pub async fn root() -> &'static str {
```

Comment:
I assume, that this is for tests only. Placing API handlers inside `lib.rs` is not good

Reply:
Yes, it is needed for testing purposes, until the actual API will be implemeted

Resolve: add additional comment to remove it later